src/components/Dropdown.jsx

```diff
@@ -30,5 +31,12 @@ class Dropdown extends React.Component {
 		);
 	}
 }
+Dropdown.propTypes = {
+	options: PropTypes.shape({
```

Comment:
We call `options.map` above, which indicates that `options` is an array, not an object.